gpii/node_modules/canopyMatchMaker/configs/kettleModuleLoader.js

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
This file is no longer required - there should be none in my branch/master already